src/components/CompanySelector.vue

```diff
@@ -0,0 +1,226 @@
+<template>
+  <div>
```

Comment:
I think that an `article` tag would be a better wrapper for this component. As you can see in the [MDN documentation](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article):

> The HTML <article> element represents a self-contained composition in a document, page, application, or site, which is intended to be independently distributable or reusable (e.g., in syndication). Examples include: a forum post, a magazine or newspaper article, or a blog entry, a product card, a user-submitted comment, an **interactive widget** or gadget, or any other independent item of content.

I highlighted the `interactive widget` part because I think that this component is an **interactive widget**.

Reply:
https://github.com/anonymouslyreportingtool/webapp/pull/7/commits/5434c3f001c00ab8dfa749491bb23be109c2b0c7 - I'll refactor one more time in the following days

Reply:
https://github.com/anonymouslyreportingtool/webapp/pull/7/commits/d4973d50d074beabfdc285a51ef87790845856f4 ✅